doc/problem/classification.md

```diff
@@ -1,26 +1,45 @@
 
-## Data Description
+## Exploratory Analysis
+
+We dispose of a labeled data set of 16 different strains of mice. Behavioral features are recorded for each mouse and each day, one example can be the time spent eating or drinking. 
 
-We dispose of a labeled data set of 16 different strains of mice. Behavioral features are recorded for each mouse and each day. One example can be the time spent eating or drinking.  
+## Data Requirement
+
+We need a set of well design behavioral features and labels for each mouse, correponding to the strain they belong.
 
 ## Methodology/ Approach Description
 
 One way to detect this relationship between genes and behaviors is to use supervised machine learning models, and see whether we can use mouse daily habit to predict its gene. If we can get high accuracy, we can conclude that there is an association between the genes and behaviors, or nature trumps nurture. In addition, we can use unsupervised machine learning models to cluster the daily mouse activities into clusters that correspond to the genes. This can highlight an even stronger relationship between genes and mouse behaviors. 
 
-## Testing Framework outline
+## Testing Framework Outline
+
+In order to perform supervised learning algorithms, we need to split the data set between a test and a training set. We fit the model using the train set and we fit the model on the test. Then, we compare the labels on the test set to define the best set of parameters for our model. The procedure has to be iterated (cross validation) in order to give confidence intervals of the errors.
+
+## Additional Remarks
 
-From our limited understanding, the results of this research might have a meaningful implication on the way we treat psychological disorders. If it turns out that nature does influence these disorders, we can probably conclude that psychological disorders is not much different than physical disabilities. Otherwise, if nature has little influence over these disorders, we can try to find way to prevent these disorders from happening. 
+From our limited understanding, the results of this research might have a meaningful implication on the way we treat psychological disorders. If it turns out that nature does influence these disorders, we can probably conclude that psychological disorders is not much different than physical disabilities. Otherwise, if nature has little influence over these disorders, we can try to find way to prevent these disorders from happening.
 
-## Initial tasks 
+Initial tasks may include:
 
 1. Clean up the existing strain_classification.py: create functions and objects.
```

Comment:
You may want to start providing anticipated names and call signatures for the functions that need to be written.  You might perhaps also add some example use case of calling those functions as well.